docs/tasks/administer-federation/cluster.md

```diff
@@ -69,6 +69,40 @@ kubefed unjoin gondor --host-cluster-context=rivendell
 You can find more details on unjoin in the
 [kubefed guide](/docs/tutorials/federation/set-up-cluster-federation-kubefed/#removing-a-cluster-from-a-federation).
 
+## Labeling Clusters
+
+You can label clusters the same way as any other kubernetes object which can help with grouping clusters and can also be leveraged by the ClusterSelector.
```

Comment:
suggested edit for capitalization and punctuation: "You can label clusters the same way as any other Kubernetes object, which can help with grouping clusters and be leveraged by the ClusterSelector."